.drone.star

```diff
@@ -69,7 +69,9 @@ config = {
 			],
 			'coverage': False
 		},
-	}
+	},
+
+	'phpstan': True
```

Comment:
phpstan runs in parallel to codestyle. I suggest move this up between codestyle and javascript sections. That will make it easier for people reading this to quickly notice all the code analysis tools that are enabled.